src/server/call_store.ts

```diff
@@ -30,7 +31,7 @@ export class CallStore {
     } else {
       try {
         let callId = msg.call_id;
-        let call = new Call(this.service, msg.call_id, msg.service_id, msg.info, this.send);
+        let call = new Call(this.service, msg.call_id, msg.service_id, msg.info, this.send, this.grpc);
```

Comment:
this feels a little ugly to have `Call` get passed a reference to the `grpc` object, but not sure that there's a better way around it, since the whole way this lib handles that grpc service is pretty convoluted